### ISSUE TITLE
Gather all BugZilla information in a bulk before running the tests

### ISSUE THREAD

**Issue by rochacbruno** [SatelliteQE/robottelo#4260]
Currently our `bz_bug_is_open` and `skip_if_bz_open` calls BZ api on demand to get the BZ status

In order to handle the WONTFIX and other resolutions

we are going to change the way we fetch information, instead of fetching BugZilla multiple times, we are going to fetch Bugzilla once before running the tests.

- [x] Grep all BZ ids found in our code 
- [x] Using conftest.py use a pre-run fixture to fetch all BZ status once and save it in a fixture object
- [ ] Change behavior of our functions to look at that list instead of calling Bz again

**Comment by rochacbruno** [comment]
@ldjebran has some ideas to tackle item 1 here: https://github.com/ldjebran/robottelo_bz_parse

**Comment by ldjebran** [comment]
yes with this we can handle any bug status, write them to files named with same status eg: CLOSED_WONTFIX.BZ.txt, CLOSED_ERRATA.BZ.txt ...

this is a very early version we can also track on wich test function they where found, and write that to a file for example:
CLOSED_WONTFIX.files.txt
that contain:
/tests/foreman/cli/test_host_collection.py::TestClass::test_function
....

**Comment by rochacbruno** [comment]
Used @ldjebran Robozilla package (pending PR https://github.com/ldjebran/robozilla/pull/3) to item 1

**Comment by rplevka** [comment]
@rochacbruno i believe this has been addressed already. Can we close this?

**Comment by rochacbruno** [comment]
@rplevka in part it has been addressed, but we still call bugzilla multiple times, we can save some time if we cache all BZ data before starting the build, but we need to discuss the impacts of doing this,

**Comment by ntkathole** [comment]
Please feel free to re-open, if this still needs fixing.
